CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 CHANGELOG
 =========
 
+unreleased
+==========
+
+* remove Django 2.2 classifier in setup.py
```

Comment:
```suggestion
* remove Django 2.2, 3.0, and 3.1 classifiers in setup.py
```

Reply:
Done in f5532518775c8c99dbced814676fe834824ae967 :)